tests/setup/sg_init.sh

```diff
@@ -12,9 +12,9 @@ set -euxo pipefail
 # ----------------------------------------------
 
 export IS_ES_SECURED=${IS_ES_SECURED:-true}
-export ES_VER=${ES_VER:-2.4.1}
-export SG_VER=${SG_VER:-2.4.1.8}
-export SG_SSL_VER=${SG_SSL_VER:-2.4.1.16}
+export ES_VER=${ES_VER:-2.4.4}
+export SG_VER=${SG_VER:-2.4.4.10}
+export SG_SSL_VER=${SG_SSL_VER:-2.4.4.19}
```

Comment:
We have been running travis tests against [these versions](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/.travis.yml#L21-L23) for some time now. So may be we can comment out travis tests agains [older versions](https://github.com/ViaQ/watches-cli/blob/2f58f73a4234ee3578bf490e79edce4231fb850c/.travis.yml#L18-L20)? We probably do not need travis run agains `4.2.2` version any loner, right?